augur/db/load/sqlite3.py

```diff
@@ -0,0 +1,52 @@
+import sqlite3
+import pandas as pd
+
+from augur.errors import AugurError
+from augur.io.metadata import Metadata
+from augur.io.sqlite3 import Sqlite3Database
+
+
+def load(metadata_file:str, metadata_table_name:str, db_file:str):
+    """Load data into a SQLite3 database file."""
+    metadata = Metadata(metadata_file)
+    database = Sqlite3Database(db_file)
+
+    load_metadata(metadata, metadata_table_name, database)
+    create_metadata_index(metadata, metadata_table_name, database)
+
+
+def load_metadata(metadata:Metadata, table_name:str, database:Sqlite3Database):
+    """Load metadata into a SQLite3 database."""
+    # Use the DataFrame head to determine column names and infer data types for
+    # table creation.
+    df_head = metadata.get_pd_df_head(n=100)
```

Comment:
Nothing strict, it was just an arbitrary value I chose.

This does have implications though, since the data types are inferred based on the retrieved values. Too little and you might end up with something like a value of `2022` for all dates, when the date column should be a string type. Too much and it becomes slow. This is very much dependent on the data, so we may need to expose this at the argparse level.

Reply:
Hmm. That's a very tricky thing for users to debug, even if we expose a knob in the CLI.

Since we're dealing with arbitrary metadata, can we always store data as text? This is basically what SQLite does under the hood anyway; they're not column types so much as type affinities and they're applied if-possible on read not write.

Reply:
Yes, storing (or at least loading) as text sounds most reasonable. I had that thought a bit after waking up this morning. It might still be easier on users to have the right data types if they will be working with the database through other means.

So the data can be first loaded as text, with the option to detect and convert into inferred types (based on all values) for users or internal functions that would benefit from having those types.

Reply:
Removed pandas-based "peeking" and instead using a `CREATE TABLE` statement with `TEXT` type for all columns: a7f322aabf0bdcd1d608282c0cd8cb2606fd53de...29196ede2e8a2d0f59defaf13c9c0a63f38c7824